beacon_node/beacon_chain/src/data_availability_checker/availability_view.rs

```diff
@@ -196,14 +195,6 @@ impl_availability_view!(
     verified_blobs
 );
 
-impl_availability_view!(
```

Comment:
Nice, looks like we may want to convert the  `impl_availability_view` macro into just a `PendingComponents` impl, after #5420.

Reply:
Yes and we should be able to get rid of the `DataAvailabilityView` entirely. The inclusion proof verification obsoletes it.  I'm guessing this is lion's next plan of action 🙂